spec/tictactoe_clojure/io_spec.clj

```diff
@@ -0,0 +1,18 @@
+(ns tictactoe-clojure.io-spec
+ 	(:require [speclj.core :refer :all]
+		[tictactoe-clojure.io :refer :all]))
+
+(describe "tictactoe-clojure.ui"
+  (describe "#display"
+  	(it "print message to the screen"
+  		(should= "Welcome to Tic Tac Toe\n"
+  			(with-out-str "1"
+  				(display "Welcome to Tic Tac Toe")))))
+
+  (describe "#prompt"
+  	(it "prints the message to the screen and ask user for input"
+  		(should= "0"
+  			(with-in-str "0"
+  				(prompt "Enter position on the board")))))
+
+  )
```

Comment:
Missing newline here. Looks like all of your other files have a newline, so let's make sure we keep this consistent. I'd also suggest moving this closing paren up a line.